src/network/serverpackethandler.cpp

```diff
@@ -147,9 +147,12 @@ void Server::handleCommand_Init(NetworkPacket* pkt)
 
 	client->net_proto_version = net_proto_version;
 
+	u16 min_proto = g_settings->getU16("protocol_version_min");
+	if(min_proto < SERVER_PROTOCOL_VERSION_MIN)
+		min_proto = SERVER_PROTOCOL_VERSION_MIN;
 	if ((g_settings->getBool("strict_protocol_version_checking") &&
 			net_proto_version != LATEST_PROTOCOL_VERSION) ||
-			net_proto_version < SERVER_PROTOCOL_VERSION_MIN ||
+			net_proto_version < min_proto ||
```

Comment:
```suggestion
	u16 min_proto = g_settings->getU16("protocol_version_min");
	if (g_settings->getBool("strict_protocol_version_checking"))
			min_proto = LATEST_PROTOCOL_VERSION;
	min_proto = rangelim(min_proto, SERVER_PROTOCOL_VERSION_MIN, SERVER_PROTOCOL_VERSION_MAX);

	if (net_proto_version < min_proto ||
```

Previously, a too high value would lock out every player. This change would also allow values like `65535` which is equivalent to the strict protocol version check.